request.go

```diff
@@ -174,6 +175,11 @@ func (r *gsRequest) prepareHTTPRequest(ctx context.Context, cfg *Config) (*http.
 // retryHTTPRequest runs & retries a HTTP request.
 // Returns UUID (string), response body ([]byte), error
 func (r *gsRequest) retryHTTPRequest(ctx context.Context, cfg *Config) (string, []byte, error) {
```

Comment:
This function contains many superfluous comments. Comments that are not necessary because they just repeat what the code is doing. Also some comments are just plain wrong, since they say something different than the code is actually doing. So please remove them from the function.